unittest/syncd/MockableSaiInterface.cpp

```diff
@@ -183,6 +183,20 @@ sai_status_t MockableSaiInterface::queryStatsCapability(
     return SAI_STATUS_SUCCESS;
 }
 
+sai_status_t MockableSaiInterface::queryStatsStCapability(
+    _In_ sai_object_id_t switch_id,
+    _In_ sai_object_type_t object_type,
+    _Inout_ sai_stat_st_capability_list_t *stats_capability)
+{
+    SWSS_LOG_ENTER();
+    if (mock_queryStatsStCapability)
```

Comment:
empty line before if

Reply:
this is similar to other functions in the file.